index.js

```diff
@@ -19,12 +19,14 @@ module.exports = function(options) {
   options = options || {};
 
   let env = process.env.BROCCOLI_ENV;
-  let projectPath = process.cwd();
+  let projectPath = options.projectPath || process.cwd();
   let projectName = getPackageName(projectPath);
+  let tsconfigPath = options.tsconfigPath || projectPath + '/tsconfig.json';
```

Comment:
The default `tsconfigPath` should be `tsconfig.tests.json` for tests (see below).